contracts/mocks/community/CommunityAdminImplementationV2Mock.sol

```diff
@@ -183,12 +183,19 @@ contract CommunityAdminImplementationV2Mock is
     }
 
     /**
-     * @notice Returns if an address is the ambassador of the community
```

Comment:
I deleted the contracts/mocks/community/CommunityAdminImplementationV2Mock.sol, so you can delete it too